config/features.rb

```diff
@@ -3,4 +3,8 @@
   feature :oai_endpoint,
           default: true,
           description: "Enable OAI-PMH endpoint for harvesting."
+
+  feature :revised_admin_set_layout,
+          default: false,
+          description: "Move the admin set to the works modal and hide the relationships tab on the deposit form"
```

Comment:
Naming is probably going to be hard for this PR.  Thanks for your first pass and here's mine.  What do you think about this for the name of the feature and description?
```suggestion
  feature :simplified_admin_set_selection,
          default: false,
          description: "The admin set to deposit works is selected from the add work modal and the relationships tab on the deposit form is hidden for depositors."
```

Reply:
@cjcolvar I had no idea what to call it, so i'm glad of the improvement :)